receiver/httpcheckreceiver/metadata.yaml

```
@@ -45,6 +45,13 @@ metrics:
       value_type: int
     unit: ms
     attributes: [http.url]
+  httpcheck.dnslookup_duration:
+    description: Measures the duration of the DNS Component of the HTTP check.
+    enabled: true
```

Please set it to false by default, and let's do a second PR to turn it on as a breaking change.
```suggestion
    enabled: false
```